src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -3268,7 +3268,7 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 		if (this._isTaskEntry(selection)) {
 			this._configureTask(selection.task);
 		} else if (this._isSettingEntry(selection)) {
-			const taskQuickPick = new TaskQuickPick(this, this._configurationService, this._quickInputService, this._notificationService, this._themeService, this._dialogService);
+			const taskQuickPick = new TaskQuickPick(this, this._configurationService, this._quickInputService, this._notificationService, this._themeService, this._dialogService, this._storageService);
```

Comment:
Can we use dependency injection here instead? `createInstance(TaskQuickPick, this)`

Reply:
yes removes a bunch of code 👍🏼